scripts/babel-relay-plugin/lib/RelayQLAST.js

```diff
@@ -145,7 +145,8 @@ var RelayQLDefinition = function (_RelayQLNode) {
   _createClass(RelayQLDefinition, [{
     key: 'getName',
     value: function getName() {
-      return this.ast.name ? this.ast.name.value : this.getType().getName({ modifiers: false }); // TODO: this.context.definitionName;
+      // TODO: this.context.definitionName;
```

Comment:
Did you intend to move this comment or was that a re-printing issue (I've seen that happen before so just checking).